public/main.js

```diff
@@ -0,0 +1,60 @@
+const randomNumber = Math.floor(Math.random() * (100) + 1)
+let counter = 0
+
+const guessNumber = () => {
+  const inputValue = document.getElementById("guessInput").value
+  if (inputValue < 1 || inputValue > 100) {
+    alert('Must be a number between 1 and 100')
+  } else if (inputValue == randomNumber) {
+    alert('You Win!')
+  } else if (inputValue > randomNumber) {
+    alert('Guess Lower')
+    counter++
+  } else if (inputValue < randomNumber) {
+    alert('Guess Higher')
+    counter++
+  }
+  document.getElementById("guesses").innerHTML = "Guesses so far: " + counter
+}
+
```

Comment:
Try splitting up your js files. This will reduce function/variable name pollution as well as getting comfortable with more module code.